source/setup/setup.rst

```diff
@@ -2,69 +2,104 @@
 Cerebro Setup
 *************************
 
-Cerebro has a few core dependencies that need to be setup before it can be used.
+We provide a Dockerfile that installs all the dependencies required by Cerebro. Running the Dockerfile will build SCALE-MAMBA and EMP-AGMPC. 
```

Comment:
Maybe you can simply say "We provide a Dockerfile that installs Cerebro as well as its dependencies."